ophyd/areadetector/plugins.py

```diff
@@ -776,10 +777,8 @@ class TransformPlugin(PluginBase, version=(1, 9, 1), version_type='ADCore'):
     )
 
 
-class FilePlugin(PluginBase, GenerateDatumInterface, version=(1, 9, 1), version_type='ADCore'):
+class FileBase:
```

Comment:
This class needs to inherit from `Device` so that the `Cpt` objects are registered (which is done via the subclass init hook).